Makefile

```diff
@@ -50,3 +50,8 @@ format:
 
 coverage:
 	$(ECHO) $(ROOT)/hack/coverage.sh --html "$(COVERAGE)" $(addprefix ./,$(DIRS))
+
+.PHONY: apiserver
+apiserver:
+	go install -v github.com/kubernetes-incubator/service-catalog/cmd/service-catalog
```

Comment:
This was started before/concurrent with all the munging of the makefiles. Perfectly willing to integrate this with how the rest of the repository behaves. Can do a followup issue/PR.

I did `go install` as it caches the zillion base k8s dependencies where `go build` does not.